client/src/components/content/VoteButtons.js

```diff
@@ -0,0 +1,22 @@
+
+import React from "react"
+
+const VoteButton = (props => {
+
+  let up = ""
+  let down = ""
+  if (props.userVote === 1) {
+    up = "upvote"
+  } if (props.userVote === -1) {
```

Comment:
I'm a bit confused here - there is only one vote value on a character? Either 1 or -1? How is it determined if 2 people vote on the same character and there are 2 downvotes for a total vote count of -2? Or is that marking a color for what the current user vote option?

Reply:
The character show router retreives and returns the user's individual vote for the individual character, if one exists.  The value of that vote, either 1 or -1, is passed to these buttons to then use css to color them appropriately.

Reply:
this is fine, just needed calrity